java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/SSL/RFC2253.java

```diff
@@ -247,9 +239,8 @@ else if (state.data.charAt(state.pos) != '\\') {
             }
         }
         //
-        // RFC 2253
-        // * (Stringchar | pair)
-        // Stringchar = <any character except one of special, "\" or QUOTATION >
```

Comment:
Just one final example, in this file, it looks like we're laying out BNF grammar rules, and again, the choice of
"1 rule per line" seems both intentional and very helpful to readability.